src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/ServiceCache.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using System;
+    using System.Collections.Immutable;
+    using Newtonsoft.Json;
+    using OpenStack.ObjectModel;
+
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ServiceCache : ExtensibleJsonObject
```

Comment:
:exclamation: This needs to be documented (along with several other elements in the same file).